Header header

# rudder status and constants
uint8 status
uint8 RDR_STATUS_UNKNOWN=0
uint8 RDR_STATUS_INIT=1
uint8 RDR_STATUS_OK=2
uint8 RDR_STATUS_FAILURE=3
uint8 RDR_STATUS_INPROGRESS=4
uint8 RDR_STATUS_COUNT=5
uint8 RDR_STATUS_INVALID=128

# rudder values and constants
uint16 target
uint16 current
uint16 RDR_VALUE_MIN=0
uint16 RDR_VALUE_INIT=180
uint16 RDR_VALUE_MAX=359
